spec/controllers/courses_controller_spec.rb

```diff
@@ -218,6 +218,22 @@
       end
     end
 
+    describe 'toggling disable student emails' do
+      it 'sets the disable student emails flag to true' do
+        expect(course.flags[:disable_student_emails]).to be_nil
+        params = { id: course.slug, course: { disable_student_emails: true } }
+        put "/courses/#{course.slug}", params: params, as: :json
+        expect(course.reload.flags[:disable_student_emails]).to eq(true)
```

Comment:
It would be slightly better if these tests use `course.disable_student_emails?` to rather than `course.flags`. That way, how `.disable_student_emails?` works is an implementation detail that could change without affecting the tests.

Reply:
Makes sense. Updated with https://github.com/WikiEducationFoundation/WikiEduDashboard/commit/a77ee30ca19af83b737810da49082f42d643f08c Note I changed the expected initial state from nil to false. Let me know if that's not right.

Reply:
Perfect. As a setting, it only makes sense for it to be either true or false and to default to false, so that's how I would have done it too.